Example/PrebidDemoJava/src/main/java/org/prebid/mobile/app/DemoActivity.java

```diff
@@ -73,37 +81,69 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             String[] wAndH = adSizeName.split("x");
             width = Integer.valueOf(wAndH[0]);
             height = Integer.valueOf(wAndH[1]);
-            if (width == 300 && height == 250) {
-                adUnit = new BannerAdUnit(Constants.PBS_CONFIG_ID_300x250, width, height);
-            } else if (width == 320 && height == 50) {
-                adUnit = new BannerAdUnit(Constants.PBS_CONFIG_ID_320x50, width, height);
-            } else {
-                adUnit = new BannerAdUnit(Constants.PBS_CONFIG_ID_320x50, width, height);
-            }
 
             enableAdditionalFunctionality(adUnit);
 
             if ("DFP".equals(adServerName)) {
-                createDFPBanner(width, height);
+                setupAndLoadAMBanner(width, height);
```

Comment:
Can you make it in the app UI that two drop downs video and video interstitial so that anyone builds that app can see the demo immediately without modifying the code?